NEWS.md

```diff
@@ -4,6 +4,7 @@
   config file into the repo at initialization. The argument defaults to 
   `options('precommit.path_cp_config_from')` to make it easy for users to use 
   their preferred hooks in every repo they initialize (#111).
+- Create `r-reticulate` env if not existant before installing into it (#114).
```

Comment:
```suggestion
- Create `r-reticulate` env if not existent before installing into it (#114).
```

Reply:
Thanks. For some reason the spell checker hook is not activated...